.github/workflows/mainpush.yaml

```diff
@@ -0,0 +1,13 @@
+name: PR Workflow
+run-name: ${{ github.actor }} pull request
+on:
+  push:
+    branches:
+      - main
+jobs:
+  TestMain:
```

Comment:
Lets dicuss how we want to format files in the project after today's daily. I agree with your remarks.

Reply:
We did decide to use `camelCase` for JavaScript / TypeScript code, but for these files it may still be better to use snake or kebab case because all of the default syntax seems to be in those styles.